### PR TITLE
Implement support for pairwise/unregistered DIDs

### LINKED ISSUE #29: Support unregistered/pairwise DID use case
Since there's no way to tell if a DID is registered or unregistered/pairwise (and this is by design), the resolution logic should be:

1. do a GET to the ledger, check if it exists
2. If doesn't exist (specifically a 404 error), then check local `didStore` to see if it's there, and return.
3. If the DID also does not exist in local did store, then re-throw the 404 error

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Closes #29.

(Ready for review. Do not merge yet, will rebase on `development` once PR #30 is merged.)

### CODE REVIEW

File: lib/methods/veres-one/veres-one.config.js [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-      'alturas',
-      'frankfurt',
-      'genesis',
-      'mumbai',
-      'saopaulo',
-      'singapore',
-      'tokyo'
-    ].map(nick => `${nick}.testnet.veres.one`)
-  }
-};
```
Comment (dlongley) [inline lib/methods/veres-one/veres-one.config.js:13]: Was this file intentionally removed?

**Review by dlongley — APPROVED** [review]
This seems ok to me ... though I wanted to make sure that the veres-one config file was intentionally removed (perhaps it's being moved to another repo?).

File: lib/methods/veres-one/veres-one.config.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/methods/veres-one/veres-one.config.js:13]: Thanks for double-checking. Yes, removed intentionally - it's not really needed in this lib.

**Comment by dmitrizagidulin** [comment]
Ok, rebased on `development`, ready to merge if nobody objects.

**Pull request merged**